modules/ROOT/pages/editor-plugin-version.adoc

```diff
@@ -1,162 +1,128 @@
-= Specify editor & plugin versions
-:description_short: Specifying editor and plugin versions for Tiny Cloud deployments.
-:description: Specifying editor and plugin versions for Tiny Cloud deployments.
+= Specify editor version & plugins
+:description_short: Specifying editor version and plugins for {cloudname} deployments.
+:description: Instructions on setting the {productname} editor version and plugins for {cloudname} deployments.
 :keywords: tinymce, cloud, script, textarea, apiKey, hybrid
 
 [[specifying-the-tinymce-editor-version-deployed-from-cloud]]
-== Specifying the TinyMCE editor version deployed from Cloud
+== Specifying the {productname} Editor Version from the Cloud
```

Comment:
```suggestion
== Specifying the {productname} editor version from {cloudname}
```
Pretty sure cloudname is a variable

Reply:
yes `cloudname: Tiny Cloud` is a variable.